www/src/pages/packages.js~HEAD

```diff
@@ -43,8 +43,9 @@ class Plugins extends Component {
                   marginRight: rhythm(3),
                   fontSize: rhythm(3/4),
                 }}>
-                Please use the search bar to find packages
-                that will make your blazing-fast site even more awesome.
+                Please use the search bar to find plugins
```

Comment:
is there a reason it's `docs/plugin-authoring/` instead of `docs/plugin-authoring.md`? Because we're directing people to a URL rather than a file in the docs folder?

Reply:
PS committed changes

Reply:
oh... yeah, it should be `/docs/plugin-authoring/` — didn't notice that error

Reply:
make sure to delete this file as well

Reply:
Like delete it on VS code and then push that change as a commit?

Reply:
Or just delete it from the commit?

Reply:
Delete it in VS Code and add that to the commit. Whoever moved the file over accidentally added this file so it needs cleaned up.

Reply:
ok done!